src/mappings/handlers/multisigCallHandler.ts

```diff
@@ -34,4 +47,117 @@ export async function handleMultisigCall(
   await Promise.all(
     accountMultisig.map((accountMultisig) => accountMultisig.save())
   );
+
+  await handleNewMultisigCall(extrinsic);
+  await callWalk.walk(extrinsic, multisigVisitor)
+}
+
+async function getTransaction(extrinsic: SubstrateExtrinsic): Promise<MultisigOperation | undefined> {
+  const { args } = extrinsic.extrinsic.method.toHuman() as unknown as MultisigArgs;
+  const { threshold, other_signatories, maybe_timepoint: timepoint, call_hash } = args;
+
+  const callIndex = extrinsic.extrinsic.method.meta.args.findIndex(arg =>
+    arg.name.toLowerCase() === 'call'
+  );
+  const call = extrinsic.extrinsic.args[callIndex];
+  const callHash = call_hash || extrinsic.extrinsic.args[callIndex]?.hash?.toHex();
+
+  if (!callHash) return;
+
+  const signer = extrinsic.extrinsic.signer.toString();
+  const multisigAddress = createKeyMultiAddress([...other_signatories, signer], threshold);
+
+  const blockCreated = timepoint ? formatStringToNumber(timepoint.height) : getBlockCreated(extrinsic);
+  const indexCreated = timepoint ? formatStringToNumber(timepoint.index) : getIndexCreated(extrinsic);
+
+  const [existingOperation] = await MultisigOperation.getByFields([
+    ['callHash', '=', callHash],
+    ['blockCreated', '=', blockCreated],
+    ['indexCreated', '=', indexCreated],
+    ['address', '=', multisigAddress]
+  ], { limit: 1 });
+
+  const operationId = existingOperation?.id || generateOperationId(callHash, multisigAddress, blockCreated, indexCreated);
+
+  const newOperation = await MultisigOperation.create({
+    id: operationId,
+    callHash,
+    status: OperationStatus.pending,
+    address: multisigAddress,
+    deposit: 1,
```

Comment:
There is a way to get deposit info from an operation?

Reply:
We can inspect events to check for changes in Reserve, however it seems like deposit should not be present on the indexer as it can be derived client-side?